tests/loggen/loggen.c

```diff
@@ -113,9 +125,10 @@ generate_message(char *buffer, int buffer_size, ThreadData *thread_context, unsi
 
   if (global_plugin_option.proxied && !thread_context->proxy_header_sent)
     {
-      str_len = generate_proxy_header(buffer, buffer_size, thread_context->index, global_plugin_option.proxy_src_ip,
-                                      global_plugin_option.proxy_dst_ip, global_plugin_option.proxy_src_port,
-                                      global_plugin_option.proxy_dst_port);
+      str_len = generate_proxy_header(buffer, buffer_size, thread_context->index,
+                                      global_plugin_option.proxy_version,
+                                      global_plugin_option.proxy_src_ip, global_plugin_option.proxy_dst_ip,
+                                      global_plugin_option.proxy_src_port, global_plugin_option.proxy_dst_port);
       thread_context->proxy_header_sent = TRUE;
       DEBUG("Generated PROXY protocol v1 header; len=%d\n", str_len);
```

Comment:
Instead of `v1` I suggest to print the actual version of proxy header.

Reply:
done